lib/broccoli/ember-app.js

```diff
@@ -369,46 +370,26 @@ class EmberApp {
     @method _initVendorFiles
   */
   _initVendorFiles() {
-    let ember = this.project.findAddonByName('ember-source');
-    let developmentEmber;
-    let productionEmber;
-    let emberTesting;
-
-    if (ember) {
-      developmentEmber = ember.paths.debug;
-      productionEmber = ember.paths.prod;
-      emberTesting = ember.paths.testing;
-    } else {
-      // in Ember 1.10 and higher `ember.js` is deprecated in favor of
-      // the more aptly named `ember.debug.js`.
-      productionEmber = `${this.bowerDirectory}/ember/ember.prod.js`;
-      developmentEmber = `${this.bowerDirectory}/ember/ember.debug.js`;
-      if (!fs.existsSync(this._resolveLocal(developmentEmber))) {
-        developmentEmber = `${this.bowerDirectory}/ember/ember.js`;
-      }
-      emberTesting = `${this.bowerDirectory}/ember/ember-testing.js`;
-    }
+    let emberSource = this.project.findAddonByName('ember-source');
+
+    assert(
```

Comment:
Theoretically, this should not be needed. Some tests were failing because we now always access `paths` on `ember-source` on line 384. This mainly provides a clearer error message during development.